kodi_addon_checker/check_repo.py

```diff
@@ -30,7 +35,7 @@ def check_repo(repo_path, parameters):
         if addon_folder[0] != '.':
             addon_path = os.path.join(repo_path, addon_folder)
             error_counter = check_addon.start(
-                error_counter, addon_path, config)
+                error_counter, addon_path, repo_addons, config)
```

Comment:
So this will work for a whole repo review, but how does it work for a single addon?